web/src/containers/StateDashboardSidebar.test.js

```diff
@@ -32,6 +36,20 @@ describe('State dashboard sidebar component', () => {
     expect(component).toMatchSnapshot();
   });
 
+  test('selects apds', () => {
+    const component = shallow(<Sidebar {...props} />);
+    const topItems = component.find('VerticalNav').prop('items');
```

Comment:
Thanks for adding this back in! (And also for teaching me that that you can target props passed to a component!)

Reply:
Honestly I only added it because I introduced a bug that broke clicking on APDs.  😬